docs/account/avatar.md

```diff
@@ -2,13 +2,12 @@
 
 #### How do I set an avatar for my account?
 
-1. Go to this site and login with your osu!droid account credentials if you haven’t logged in yet.
+1. Go to [this](https://osudroid.moe/user/?action=login) site and login with your osu!droid account credentials if you haven’t logged in yet.
 2. Open the left-side drawer and click on Profile.
-3. Press “Select file” and select an image file to upload as avatar. Keep in mind that there is a file size limitation of 1 MB.
+3. Press “Select file” and select an image file to upload as avatar. Keep in mind that there is a file size limitation of around 1.5 MB.
 4. Crop your image using the tools provided if necessary.
 5. Save your image by clicking on the blue checkmark button on the bottom-right side of your screen.
-6. Press “Upload” to finally upload your avatar.
-7. Clear the game’s cache via system settings. The methods to clear an application’s cache can differ for every phone. Hit Google if you don’t know how.
+6. Press “Upload” to upload your avatar.
```

Comment:
```suggestion
6. Press “Upload” to upload your avatar.
*NOTE: If you don't see your avatar change on the website. Don't panic, it just mean that the browser (and the game) saved your old avatar to the cache. You can refer to [here](https://www.xda-developers.com/how-clear-app-cache-android/#:~:text=Within%20the%20app%20settings%2C%20look,Delete%20app%20data%20options%20here.) to know how to do so on Android*
```
From my own test, user still have to clear cache to see their avatar change (both on the web and the app)

Reply:
Weird, that's not how 304s work, and you should not need to clear the cache [as per 1.6.8](https://github.com/osudroid/osu-droid/blob/0bf8deedbf01aa2226a91944d7ae6ed2487143c0/assets/app/changelog.md#changes).

I guess this can be kept until I get around to investigate in-game behavior.

Reply:
Weird, that's not how 304s work, and you should not need to clear the cache [as per 1.6.8](https://github.com/osudroid/osu-droid/blob/0bf8deedbf01aa2226a91944d7ae6ed2487143c0/assets/app/changelog.md#changes).

I guess this can be kept until I get around to investigate in-game behavior.